src/hooks/usePlaceListPois.ts

```diff
@@ -5,26 +5,20 @@ import Places from "../api/Places"
 import { AggregatePlaceAttributes } from "../entities/Place/types"
 import { getPois } from "../modules/pois"
 
-export function usePlaceListPois(options?: {
-  limit: number
-  offset: number
-  positions?: string[]
-}) {
+export function usePlaceListPois() {
```

Comment:
This should handle `offset` and `limit` the same way that `usePlaceListMyFavorites` does